python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -655,6 +655,50 @@ def rolling_avg(val, avg):
         kwargs.update({"chunks": offsets})
         return grouped_values.apply_chunks(function, **kwargs)
 
+    def transform(self, function):
+        """Apply an aggregation, then broadcast the result to the group size.
+
+        Parameters
+        ----------
+        function: str or callable
+            Aggregation to apply to each group. Note that currently,
+            only aggregations are supported by `transform`.
```

Comment:
I pulled this locally and tried several inputs. It seems for callables only form `lambda x: x.agg` is accepted, should we make it clear in the docstring?

Reply:
I added a note that hopefully clarifies this. Let me know if you would add/rephrase anything.

Reply:
So currently it redirects the user to look up for `agg`, and that's ok. I think there are two things we can improve: we can add a `see also` section here for easy redirection. But more importantly in `agg` docstring the description for `func` should be clearer. The latter is out of the scope of this PR, I think.